pokemon_ai/classes/pokemon/pokemon.py

```diff
@@ -13,6 +15,8 @@ class Pokemon:
     All attributes related to a Pokemon.
     """
 
+    STRUGGLE = Move(name="Struggle", base_damage=50, pp=1, type=PokemonType.TYPELESS, is_special=True, base_heal=0, status_inflict=None)
```

Comment:
Struggle [is a normal type](https://bulbapedia.bulbagarden.net/wiki/Struggle_(move))